java/src/jmri/jmrit/operations/automation/actions/RunTrainAction.java

```diff
@@ -65,7 +65,11 @@ public void doAction() {
             if (InstanceManager.getDefault(TrainCustomManifest.class).doesCommonFileExist()) {
                 log.warn("Manifest CSV common file exists!");
             }
-            InstanceManager.getDefault(TrainCustomManifest.class).addCVSFile(train.createCSVManifestFile());
+            if (!InstanceManager.getDefault(TrainCustomManifest.class).addCVSFile(train.createCSVManifestFile())) {
```

Comment:
No need to fix. But I noticed that the method name is `addCVSFile` and not `addCSVFile`.